### PR TITLE
ref: Remove references to MongoDB

### PR DESCRIPTION
**Pull request by lgarber-akamai**
## 📝 Description

This change removes all references to MongoDB from both resource implementations and documentation.

### CODE REVIEW

**Comment by lgarber-akamai** [comment]
/acctest sha=e3e509efc0dbd37f65aa9f463272af779104eca5 pkg=linode/...

**Review by zliang-akamai — APPROVED** [review]

File: linode/databasepostgresql/schema_datasource.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ var dataSourceSchema = map[string]*schema.Schema{
 	"database_id": {
 		Type:        schema.TypeInt,
 		Required:    true,
-		Description: "The ID of the MongoDB database.",
+		Description: "The ID of the PostgreSQL database.",
```
[the remark this answers is not in the document]
Comment (lgarber-akamai) [reply linode/databasepostgresql/schema_datasource.go:11]: Yeah LOL

**Comment by yec-akamai** [comment]
Approved pending on fixing the integration tests. Not able to see why it's failing from my view...

**Comment by zliang-akamai** [comment]
> Approved pending on fixing the integration tests. Not able to see why it's failing from my view...

The test failure seems irrelevant to this PR and is being fixed [here](https://github.com/linode/terraform-provider-linode/pull/811)

**Comment by yec-akamai** [comment]
> > Approved pending on fixing the integration tests. Not able to see why it's failing from my view...
> 
> The test failure seems irrelevant to this PR and is being fixed [here](https://github.com/linode/terraform-provider-linode/pull/811)

Thank you for the pointer!

**Comment by zliang-akamai** [comment]
/acctest sha=5d7e504cc7305f272bc04659dfa615036855e45e pkg=linode/...

**Comment by zliang-akamai** [comment]
/acctest sha=5d7e504cc7305f272bc04659dfa615036855e45e pkg=linode/stackscript

**Pull request merged**